tests/storage/restore.go

```diff
@@ -828,6 +828,51 @@ var _ = SIGDescribe("VirtualMachineRestore Tests", func() {
 				return dv
 			}
 
+			verifyOwnerRef := func(obj metav1.Object, apiVersion, kind, name string, uid types.UID) {
+				ownerRefs := obj.GetOwnerReferences()
+				Expect(ownerRefs).To(HaveLen(1))
+				ownerRef := ownerRefs[0]
+				Expect(ownerRef.APIVersion).To(Equal(apiVersion))
+				Expect(ownerRef.Kind).To(Equal(kind))
+				Expect(ownerRef.Name).To(Equal(name))
+				Expect(ownerRef.UID).To(Equal(uid))
+			}
+
+			verifyRestore := func(restoreToNewVM bool, originalDVName string) {
+				if restoreToNewVM {
+					checkNewVMEquality()
+					Expect(restore.Status.DeletedDataVolumes).To(HaveLen(0))
+				} else {
+					Expect(restore.Status.DeletedDataVolumes).To(HaveLen(1))
+					Expect(restore.Status.DeletedDataVolumes).To(ContainElement(originalDVName))
+					_, err = virtClient.CdiClient().CdiV1beta1().DataVolumes(util.NamespaceTestDefault).Get(context.Background(), originalDVName, metav1.GetOptions{})
+					Expect(errors.IsNotFound(err)).To(BeTrue())
+				}
+
+				restores := restore.Status.Restores
+				Expect(restores).To(HaveLen(1))
+
+				pvcName := restores[0].PersistentVolumeClaimName
+				Expect(pvcName).ToNot(BeEmpty())
+				pvc, err := virtClient.CoreV1().PersistentVolumeClaims(util.NamespaceTestDefault).Get(context.Background(), pvcName, metav1.GetOptions{})
+				Expect(err).ToNot(HaveOccurred())
+
+				targetVM := getTargetVM(restoreToNewVM)
+
+				if libstorage.IsDataVolumeGC(virtClient) {
+					verifyOwnerRef(pvc, targetVM.APIVersion, targetVM.Kind, targetVM.Name, targetVM.UID)
```

Comment:
maybe should add a wait before this to make sure the dv is deleted cause if not the ref will fail, it might be racy